people/methods/tag.js

```diff
@@ -0,0 +1,51 @@
+var techwords = require('./tagwords');
+var con = require('./dbConnection');
```

Comment:
pass only using req.app.get this is not a correct way you are creating a lot of coupling with db connections